### ISSUE TITLE
Warn when publishing a package with too short description

### ISSUE THREAD

**Issue by alexeyinkin** [dart-lang/pub#3526]
Package rating suffers if one publishes a package with a too short description:
https://pub.dev/packages/model_fetch/versions/0.3.0/score

But there is no warning when such package is published. A warning would remind the author to fix this and spare a one vain publication.

**Comment by sigurdm** [comment]
Yeah, doing an extra publication just to make a longer description sucks.

Different servers might impose different limits on the description length.
Also we might change the limit on the server side.

Therefore it would be a shame to bake the limit into the client.

Hopefully this problem will go away once we get staged publishing. https://github.com/dart-lang/pub/issues/2795 . Then you will be able to get the server's analysis before "going live".

**Comment by sigurdm** [comment]
wontfix
